.github/workflows/vizier_release.yaml

```diff
@@ -87,4 +90,27 @@ jobs:
         gh release create "${TAG_NAME}" --title "Vizier ${TAG_NAME#release/vizier/}" \
           --notes $'Pixie Vizier Release:\n'"${changelog}"
         gh release upload "${TAG_NAME}" vizier-artifacts/*
+        url="$(gh release view "${TAG_NAME}" -q '.url' --json url)"
+        echo "url=${url}" >> $GITHUB_OUTPUT
       # yamllint enable rule:indentation
+  update-readme:
+    name: Update README with latest release
```

Comment:
should we gate this on non-RC builds? `if: ${{ !contains(github.event.ref, '-') }}`

Reply:
needs takes care of it.